draft-ietf-anima-brski-prm.md

```diff
@@ -1103,589 +1166,583 @@ pledgeenrollrequesttrigger = {
-Note, the registrar is already aware that the bootstrapping is performed in a pledge-responder-mode due to the use of the EE (RegAgt) certificate for TLS and the provided PVR as JSON-in-JWS object.
+A successful interaction with the Key Infrastrcutre will result in a pledge LDevID certificate.
+The registrar MUST reply to the Registrar-Agent with the Enroll-Response (Enroll-Resp) as defined in {{er-artifact}} in the body of a 200 OK response.
+In the response header, the Content-Type field MUST be set to `application/pkcs7-mime`.
 
-* If the registrar receives a PER with Content-Type header: `application/jose+json`, it MUST verify the wrapping signature using the certificate indicated in the JOSE header.
+If the domain registrar is unable to return the Enroll-Resp, it SHOULD respond with an HTTP server error code.
+The following server error codes SHOULD be used:
 
-* The registrar verifies that the pledge's certificate (here IDevID), carried in "x5c" header field, is accepted to join the domain after successful validation of the PVR.
+* TODO
 
+Note that while BRSKI-PRM targets the initial enrollment, re-enrollment may be supported in a similar way with the exception that the current pledge LDevID certificate is used instead of the IDevID certificate to sign the PER artifact (see also {{tper}}).
 
-### Enroll Pledge by Domain CA (backend interaction)
 
-If both succeed, the registrar utilizes the PKCS#10 request contained in the JWS object body as "P10" parameter of "ietf-sztp-csr:csr" for further processing of the Enroll-Request with the corresponding domain CA.
-It creates a Registrar Enroll-Request (RER) by utilizing the protocol expected by the domain CA.
+### Request Artifact: Pledge Enroll-Request (PER)
 
-The domain registrar may either directly forward the provided PKCS#10 request to the CA or provide additional information about attributes to be included by the CA into the requested LDevID certificate.
+Identifical to the PER artifact defined in {{per-artifact}}.
+The Registrar-Agent MUST NOT modify PERs received from pledges.
 
-The approach of sending this information to the CA depends on the utilized certificate management protocol between the RA and the CA and is out of scope for this document.
 
+### Response Artifact: Registrar Enroll-Response (Enroll-Resp) {#er-artifact}
 
-### Response Artifact: Enroll-Response (Enroll-Resp)
+The Enroll-Response (Enroll-Resp) artifact SHALL be an authenticated self-contained object signed by the domain CA, containing a pledge LDevID certificate.
 
-The registrar SHOULD respond with an HTTP 200 OK in the success case or fail with HTTP 4xx/5xx status codes as defined by the HTTP standard.
+For this specification, the Enroll-Resp artifact MUST be a certs-only CMC Simple PKI Response (PKCS#7) as defined in {{Section 4.1 of !RFC5272}} (following EST {{RFC7030}}).
```

Comment:
This is why we need the RFC5272 reference, as the format is not defined in EST.

Reply:
Okay, thanks for the clarification